tools/docker/Dockerfile

```diff
@@ -51,8 +51,15 @@ RUN apt-get -qqy install \
 
 RUN apt-get -y autoremove
 
-RUN pip install --upgrade pip
-RUN pip install virtualenv
+RUN pip2 install --upgrade pip
+RUN pip2 install virtualenv
+
+RUN pip3 install --upgrade pip
+RUN pip3 install virtualenv
+
+# Ensure we have up-to-date six
```

Comment:
Where do we rely on `six` without using a virtualenv (either through `wpt` or `tox`)? If six is now required globally, we'd need to update our docs. Furthermore, we even have `six` checked into `third_party`?

Reply:
Let's file an issue. Things like this are very easy to fall through and contributors will start to assume `six` is available by default sooner or later (as most people who do Python development already have `six` installed globally anyway, and this change will make `six` available on Taskcluster), until one day some user who doesn't work on Python much complains that our docs are wrong.

Also how is the decision task passing now if it already uses `six`? `Dockerfile` doesn't include `six` at the moment; do we get it through a transient dependency somehow?

Reply:
Are you still going to fix this in this PR? If not, could you file an issue? IIUC, this can be an easy fix; we can essentially reimplement `six.ensure_*` in `localpaths.py`.

Reply:
I actually fixed the underlying issue in this PR; we could probably remove six from the docker image now.